core/gethwrappers/go_generate.go

```diff
@@ -3,7 +3,7 @@
 package gethwrappers
 
 // Make sure solidity compiler artifacts are up to date. Only output stdout on failure.
-//go:generate ./generation/compile_contracts.sh
```

Comment:
typo?

Reply:
Ah yep, definitely a thing I did locally to reduce the time to generate the wrappers 😄